cross/x86_64-binutils/Portfile

```diff
@@ -24,10 +24,14 @@ if {$subport eq $name} {
 foreach ostarget {linux dragonfly freebsd netbsd openbsd} {
     subport x86_64-${ostarget}-binutils {
         crossbinutils.setup     x86_64-${ostarget} ${version}
+        depends_lib             port:$name
         configure.args-append   --disable-werror
         if {${ostarget} eq "linux"} {
             depends_build-append    port:bison
         }
+        post-destroot {
+            delete ${destroot}${prefix}/share/doc/x86_64-binutils
+        }
```

Comment:
Why delete the docs? (Not complaining, just asking.)

Reply:
because the "mother" port x86_64-binutils installs the docs for all subports. That is all, what x86_64-binutils does.

Reply:
Put in a comment maybe explaining that.